### PR TITLE
add a dump of all glottolog's data

### PR DESCRIPTION
**Pull request by camoverride**
I think it makes sense to add a csv dump of the glottolog data to this repo. Below are my reasons:

- The data is already here, but not saved in csv format.
- The output csv is only 4.2 MB and 25852 lines.
- Having a csv here makes it __very__ obvious what glottolog has to offer.
- The data already lives in [the glottolog cldf](https://github.com/glottolog/glottolog-cldf/blob/master/cldf/languages.csv) but this is not obvious and took some digging.
- The data can be generated programmatically with the [pyglottolog repo](https://github.com/glottolog/pyglottolog), but this requires downloading this repo anyway.

In this MR, I simply added a csv dump of the data that I generated with [the glottolog python library](https://github.com/glottolog/pyglottolog) and updated the README accordingly.

Additionally, it would make sense to set up CI/CD to automatically generate a new CSV whenever the version # of this repo is bumped, but that will take a little work.

tl;dr - it makes more sense to have the data in csv format here instead of requiring a separate python library to generate it.

Please let me know if this makes sense :) It would help me personally and I imagine it would be useful and convenient for others.

### CODE REVIEW

**Comment by xrotwang** [comment]
You are hitting a sore point here :) You are right that finding released version of the data in a more usable format shouldn't require much digging - in particular from here. However, I think adding this CSV here is not my favourite solution.
- First, there's the issue of syncing the "raw" data and the derived version. While it may be technically possible to make sure the CSV reflects the status of the repos at each commit, this is not what we want. We want people to use released, citeable versions and not intermediate states of the data. If this CSV is only regenerated upon releasing OTOH, this can easily lead to confusion if it is kept at the same place as the "raw" data - even if we'd put in a `dist/` directory or similar.
- Second, we already have a couple of ways to distribute release data:
  - the web app https://glottolog.org - for a browseable version,
  - various download formats available through https://glottolog.org/meta/downloads
  - the Glottolog CLDF dataset (released on Zenodo)
  I wouldn't want to add yet another half-specified distribution format that people will come to rely on.
- Third, the data in this CSV isn't really "all Gottolog data", e.g. the references and relations between languages and sources are missing, etc. So this would only be a halfway solution, yet creating an expectation that we'd want to satisfy in the future, too.

So, I'm all for making it clearer how the data should be accessed - and doing that in the README here.

Ideally, going forward, I'd like to make the Glottolog CLDF the authoritative version - but some of the formats on Glottolog's download page have regular users, so we'll keep on providing these as well. Trying to balance stability and maintainability here is tricky.

**Comment by xrotwang** [comment]
@camoverride I'll keep this PR open until the README here is in a better state - hopefully one that you'd also be happy with.

**Comment by camoverride** [comment]
@xrotwang - gotcha, that all makes sense and sounds very reasonable. In that case, consider this issue to instead be __include information about accessing data in README__.

When I first discovered glottolog, I was directed to this repo and had to spend time digging before I actually found the csv I was looking for. So I think that adding these to the README would be very useful:

- a link to the csv / mention of [glottolog cldf](https://github.com/glottolog/glottolog-cldf/blob/master/cldf/languages.csv)
- a description of the data / the column headers

:)
